Civi/WorkflowMessage/GenericWorkflowMessage/alex.ex.php

```diff
@@ -0,0 +1,79 @@
+<?php
+return [
+  "tags" => [],
```

Comment:
Just noting that we have been removing unnecessary double quotes -maybe these could be formatted out

Reply:
Yea it trigers my OCD a bit too, but the reality is that PHP doesn't care, and I've been consistently hearing the advice "don't micro-optimize it's a waste of your time" from the wider dev community. So I try to grit my teeth when I see them and move on.

Reply:
@colemanw it's a bit more than OCD because when we do use quotes correctly that ide notification can help us find real bugs. In this case it's new code & probably just a case of getting @totten to adjust his IDE settings or get into the habit

Reply:
In this case, the example originally was done in JSON (which required double-quotes - because JSON cares). But doing pure-JSON for the example created an extra layer of complexity when I went to define examples that build-on other examples (e.g. the case-activity example builds on the alex example). So I converted it to PHP. And, as you say, PHP doesn't care, so I just left the double-quotes...

Not hard to change though...

Reply:
Latest rebase changes single/double quotes.